docs/supportedshaders.md

```diff
@@ -0,0 +1,41 @@
+> ### Q. What shaders have been tested and are working with Iris? 
+
+#### A. The following shaders have been tested and work well:
+
+• [Complementary Shaders](https://www.curseforge.com/minecraft/customization/complementary-shaders)
```

Comment:
It would be better to use Markdown bullet points (- or *) instead of the actual bullet point character for the purposes of formatting on GitHub.

Reply:
switched :)